Dockerfiles/Dockerfile.webui

```
@@ -11,19 +11,26 @@ WORKDIR /app/apps
+ARG TLDW_INTERNAL_API_ORIGIN=http://app:8000
 
 ENV NEXT_PUBLIC_API_URL=${NEXT_PUBLIC_API_URL}
+ENV NEXT_PUBLIC_API_BASE_URL=${NEXT_PUBLIC_API_BASE_URL}
 ENV NEXT_PUBLIC_API_VERSION=${NEXT_PUBLIC_API_VERSION}
 ENV NEXT_PUBLIC_X_API_KEY=${NEXT_PUBLIC_X_API_KEY}
+ENV NEXT_PUBLIC_TLDW_DEPLOYMENT_MODE=${NEXT_PUBLIC_TLDW_DEPLOYMENT_MODE}
+ENV TLDW_INTERNAL_API_ORIGIN=${TLDW_INTERNAL_API_ORIGIN}
 
-RUN npm --workspace tldw-frontend run build
+RUN node /app/apps/tldw-frontend/scripts/validate-networking-config.mjs
+RUN cd /app/apps/tldw-frontend && bun run build
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)